akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -41,6 +47,7 @@ class AskSpec extends TestKit("AskSpec") with TypedAkkaSpec with ScalaFutures {
     "must fail the future if the actor is already terminated" in {
       val ref = spawn(behavior)
       (ref ? Stop).futureValue
+      TestProbe().expectTerminated(ref, 1.second)
```

Comment:
why 1.second, isn't there a default?
1 can be too small, and for no reason

Reply:
Yeah, good point, didn't think about it.